### PR TITLE
PD-376 render chart data into correct format and display charts

### PR DESCRIPTION
**Pull request by tobyhede**
YOLO

### CODE REVIEW

File: app/decorators/widget_decorator.rb [hunk 1]
```diff
@@ -26,6 +39,10 @@ def summary
     end
   end
 
+  def last_updated
```
Comment (elisechant) [inline app/decorators/widget_decorator.rb]: totally onboard with the need to "re think" whether we need this stuff. later.

**Comment by elisechant** [comment]
@tobyhede assume this is just a 1:1 mapping from productotype

File: config/initializers/npm.rb [hunk 2]
```diff
@@ -1 +0,0 @@
-# system 'npm install'
```
Comment (elisechant) [inline config/initializers/npm.rb]: ouch

File: spec/decorators/widget_decorator_spec.rb [hunk 3]
```diff
@@ -23,60 +23,367 @@
...
+      describe 'datapoint' do
+        subject(:datapoint) { chart_dataset['data'].first }
+        it { is_expected.to include('label') }
+        it { is_expected.to include('value') }
+      end
+    end
 
   end
 
+  {
```
Comment (elisechant) [inline spec/decorators/widget_decorator_spec.rb]: could we have this in a fixtures/ json file? going to need the same one for front end tests

File: spec/serializers/widget_serializer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe WidgetSerializer, type: :serializer do
+
+  let(:widget) { FactoryGirl.create(:widget_with_datasets) }
+  let(:serializer)  { WidgetSerializer.new(widget, :include => 'datasets.datapoints') }
+
+  subject(:data)    { JSON.parse(serializer.to_json) }
+
+  it { is_expected.to include('id') }
```
Comment (elisechant) [inline spec/serializers/widget_serializer_spec.rb]: include and check type?

File: config/initializers/npm.rb (hunk 2 above)
Comment (tobyhede) [inline config/initializers/npm.rb]: I thought I deleted that file. Good call.

Comment (tobyhede) [inline config/initializers/npm.rb]: Oh I did. 
I made a new way of starting the server that runs npms and stuff.

Comment (elisechant) [inline config/initializers/npm.rb]: k. consider close then.

**Pull request merged**